system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -492,6 +495,29 @@
 
     (index/wait-until-indexed)
 
+    (testing "reversing usage_score reverses sort-order"
```

Comment:
Why don't you use the existing functions to search and compare sorting orders? I like the old way where it lists the collections and the order explicitly. I might be missing something, but I don't see any benefit in doing this test in a different way from before.